src/bci_build/package/apache_tomcat.py

```diff
@@ -91,19 +90,14 @@ def _get_sac_supported_until(
             Replacement(
                 regex_in_build_description="%%tomcat_version%%", package_name=tomcat_pkg
             ),
-            Replacement(
-                regex_in_build_description="%%tomcat_minor%%",
-                package_name=tomcat_pkg,
-                parse_version=ParseVersion.MINOR,
-            ),
         ],
         cmd=[
             f"/usr/{'libexec' if os_version in (OsVersion.TUMBLEWEED, OsVersion.SLE16_0) else 'lib'}/tomcat/server",
             "start",
         ],
         exposes_tcp=[8080],
         env={
-            "TOMCAT_MAJOR": tomcat_major,
+            "TOMCAT_MAJOR": math.floor(tomcat_ver),
```

Comment:
nit: `int(tomcat_ver)` achieves the same thing and does not require an extra library import.